src/components/app/types.ts

```diff
@@ -102,7 +103,7 @@ export interface AppDetails extends CDModalProps {
     appId: number
     deploymentAppType?: DeploymentAppTypes
     externalCi?: boolean
-    userApprovalConfig?: string
+    approvalConfigData?: ApprovalConfigDataType
```

Comment:
Please check if its feasible to have this key in app details since we poll it and Can sync with prakhar since he was working on block state api

Reply:
Pending on Gireesh to confirm this